### PR TITLE
Add toolbar top tests

### PR DESCRIPTION
**Pull request by marthacryan**
Adds tests for:
- Undo
- Redo
- Clear
- Steps
- Help button
- Search button

### CODE REVIEW

File: tests/streamlit_ui_tests/toolbar/top.spec.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        // Clear the analysis
+        await mito.getByTitle('Removes all of the transformations you\'ve made to imported dataframes.').click();
+        await expect(mito.getByText('Clear your current analysis?')).toBeVisible();
+        await mito.getByText('Clear', { exact: true }).click();
+
+        // Expect that the column does not exist
+        await expect(mito.locator('.endo-column-header-final-text', { hasText: /new-column/ })).not.toBeVisible();
+    });
+
+    test('Steps', async ({ page }) => {
```
Comment (aarondr77) [inline tests/streamlit_ui_tests/toolbar/top.spec.ts:40]: Do you want to test that we can:
1. check out a previous step by clicking on it in the taskpane
2. see the popup in the left hand corner that tells us to go back to the end of the analysis
3. clicking on the button in the left hand corner takes us back to the end of the analysis 


.. we could also do this in a separate taskpane test though

File: tests/streamlit_ui_tests/toolbar/top.spec.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        await expect(mito.getByText('Added column', { exact: true })).toBeVisible();
+    })
+
+    test('Help button', async ({ page }) => {
+        const mito = await getMitoFrameWithTestCSV(page);
+
+        const popupPromise = page.waitForEvent('popup');
+        await mito.getByText('Help').click();
+        const popup = await popupPromise;
+        await expect(popup.url()).toBe('https://discord.com/invite/XdJSZyejJU');
```
Comment (aarondr77) [inline tests/streamlit_ui_tests/toolbar/top.spec.ts:58]: Sweet!

File: tests/streamlit_ui_tests/toolbar/top.spec.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    });
+
+    test('Search button', async ({ page }) => {
```
Comment (aarondr77) [inline tests/streamlit_ui_tests/toolbar/top.spec.ts:61]: Can we do a full screen test too?

**Review by aarondr77 — COMMENTED** [review]
Tiny comments and then LGTM

File: tests/streamlit_ui_tests/toolbar/top.spec.ts (hunk 3 above)
Comment (marthacryan) [reply tests/streamlit_ui_tests/toolbar/top.spec.ts:61]: I tried but I wasn't quite sure how to test that. Any ideas?

**Pull request merged**